sdk/lib/opentelemetry/sdk/resources/resource.rb

```diff
@@ -21,7 +21,7 @@ class << self
           # @return [Resource]
           def create(labels = {})
             frozen_labels = labels.each_with_object({}) do |(k, v), memo|
-              raise ArgumentError, 'label keys and values must be strings' unless k.is_a?(String) && v.is_a?(String)
+              return new({}.freeze) unless k.is_a?(String) && v.is_a?(String)
```

Comment:
Same question here. This is done as configuration, though, so technically it's OK to keep the stricter raising check rather than the defaulting check. I guess this could blow up below if someone passed `false` as a value, for example. 🤷‍♂ opinions?

Reply:
Given that this *is* called at configuration time, and the error handling spec suggests raising for configuration errors, I think I should revert this change and stick with eagerly raising. Would you be OK with that @mwear ?